app/views/self_service/advisers/_advisers_table.html.erb

```diff
@@ -22,6 +23,9 @@
               <%= link_to adviser.name, edit_self_service_firm_adviser_path(adviser.firm, adviser), class: 't-edit-link' %>
             </td>
             <td class="t-adviser-postcode" data-dough-filterable><%= adviser.postcode %></td>
+            <td>
+              <%= link_to 'Remove from Directory', self_service_firm_adviser_path(firm, adviser), method: :delete, data: {confirm: 'Are you sure?'}, :class => 'button button--small t-remove-button' %>
```

Comment:
We've made use of rails `button_to` method which builds a form in the background and created a dough component, `ConfirmableForm` that can be reused later. Thanks for the pointers